### DISCUSSION TITLE
Import "browser" could not be resolved

### DISCUSSION THREAD

**Discussion by thewh1teagle** [brython-dev/brython#2446] [Q&A] [answered]
I use Brython inside `.py` files in `VSCode` and intellisense shows me many errors, it doesn't detect Brython and I don't have autocomplete at all.
Is it possible to get autocompletion inside `.py` files in `VSCode` when writing with Brython?

Also, how can I print javascript object from Brython in a way that the console will show it as object and not as the string `javascript object`

**Comment by PierreQuentel** [comment, marked as answer]
The first point has been discussed in issue #1938. Support is possible but no one seems to want to write the code for it.

For the second one:
```python
from browser import window, console

console.log(window.js_obj)
```

**Reply by thewh1teagle** [reply]
Thanks!
the `browser.console.log` useful and works great.
